src/main/java/HTTP/HTTPResponseDTO.java

```diff
@@ -0,0 +1,29 @@
+package HTTP;
+
+import org.json.JSONObject;
+
+public class HTTPResponseDTO {
+
+    public String status;
```

Comment:
It is a bit unclear why do you save status code of the response

Reply:
I'll validate the status of response is correct (200 or 201) and only after that, I will validate the response body in further tests.